test/functional/plugin/health_spec.lua

```diff
@@ -80,36 +80,72 @@ describe('health.vim', function()
 
   describe(":checkhealth", function()
     it("concatenates multiple reports", function()
-      command("checkhealth success1 success2")
+      command("checkhealth success1 success2 lua_success")
       helpers.expect([[
 
-        health#success1#check
+        success1: health#success1#check
         ========================================================================
         ## report 1
           - OK: everything is fine
 
         ## report 2
           - OK: nothing to see here
 
-        health#success2#check
+        success2: health#success2#check
         ========================================================================
         ## another 1
           - OK: ok
+
+        lua_success: v:lua.require("health").check("test_plug.health.lua_success")
```

Comment:
this a little weird, I don't think lua checks should be marked as "secondary" to vimL functions. perhaps just `require("test_plug.health.lua_success")` which is equivalent of the autoload file references (which not are complete calls).